### PR TITLE
Implements LIHEAP eligibility logic.

### PR DESCRIPTION
**Pull request by rajivrajiv**
Implemented liheapEligible() and tested it.

### CODE REVIEW

File: src/site/static/eligibility.liquid [hunk 1]
```diff
@@ -2122,10 +2122,46 @@ pageClass: "page-public-assistance"
       underIncomeLimit);
   }
 
+  /*
   function liheapEligible() {
     // TODO
     return false;
   }
+  */
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2130]: Oops, leftover comment here.

File: src/site/static/eligibility.liquid [hunk 2]
```diff
@@ -2122,10 +2122,46 @@ pageClass: "page-public-assistance"
...
+      2700.17,
+      3531.0,
+      4361.83,
+      5192.75,
+      6023.59,
+      6854.43,
+      7010.21,
+      7166.00,
+      7321.78,
+      7447.56
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2150]: Should be `7477.56`

File: src/site/static/eligibility.liquid [hunk 3]
```diff
@@ -2122,10 +2122,46 @@ pageClass: "page-public-assistance"
...
+    //   https://www.acf.hhs.gov/sites/default/files/documents/ocs/COMM_LIHEAP_Att1SMITable_FY2023.pdf
+    const grossLimit = new MonthlyIncomeLimit([
+      2700.17,
+      3531.0,
+      4361.83,
+      5192.75,
+      6023.59,
+      6854.43,
+      7010.21,
+      7166.00,
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2148]: Either `.00` like this or `.0` as above in `3531.0` is fine. Can you choose one for consistency?

**Review by trevorshannon — CHANGES_REQUESTED** [review]
Thanks very much!  These comments should be pretty quick to resolve.

**Review by trevorshannon — APPROVED** [review]
Looks good!  Feel free to merge whenever convenient.

**Comment by rajivrajiv** [comment]
Thanks.

**Pull request merged**